### DISCUSSION TITLE
Multiple files/images request and only 1 file being sent

### DISCUSSION THREAD

**Discussion by NebilSahin** [johannschopplich/nuxt-api-party#59] [Q&A]
I have been using "Nuxt Api Party" in my project for a while now, but I have encountered a problem with sending multiple images.

The problem occurs whenever I try to send multiple images. The "Nuxt Api Party" form data serialises my Formdata and only adds the first image in my request and disregards the rest as shown in the screenshot below: 

![image](https://github.com/johannschopplich/nuxt-api-party/assets/61326587/b44b40b5-c1d9-4415-ada6-823ec92b8cb8)

even though I have added two images (below is the console log for Formdata values):
![image](https://github.com/johannschopplich/nuxt-api-party/assets/61326587/fff0d919-5634-4812-b9f9-808bf207ff0b)

I have been stuck on this problem for awhile now, is there a way to send multiple images or files ?
Any help is much appreciated, thanks.

**Comment by johannschopplich** [comment]
Hi there @NebilSahin, can you please test the latest version of Nuxt API Party? Multiple form data values weren't supported, which is now fixed.

Please install [v0.21.3](https://github.com/johannschopplich/nuxt-api-party/releases/tag/v0.21.3). Feedback welcome if that solves your problem. Thanks!

**Comment by zAlweNy26** [comment]
Hi, I'm still facing this issue when uploading multiple files looping through an array. How can I fix this?

**Reply by johannschopplich** [reply]
Can you please provide a reproducible example and open an issue? Thanks in advance!
